src/LTOCli/CLI.py

```diff
@@ -59,6 +59,13 @@ def main():
     parser_lease = subparsers.add_parser('lease', help="Create a Lease Transaction, type 'lto lease --help' for more information")
     lease_subparser = parser_lease.add_subparsers(dest='subparser-name-lease')
 
+    parser_lease_list = lease_subparser.add_parser('list', help="Returns the list of leasing that the user has conceded")
+    parser_lease_list.add_argument('--account', type=str , nargs=1, required=False, help="Use this option to select one of the account previously stored, The account can be referenced by name or address, if this option is omitted, the default account is used")
+
+    parser_lease_list_inbound = lease_subparser.add_parser('list-inbound', help="Returns the list of leasing in favor of the user")
+    parser_lease_list_inbound.add_argument('--account', type=str , nargs=1, required=False, help="Use this option to select one of the account previously stored, The account can be referenced by name or address, if this option is omitted, the default account is used")
```

Comment:
*one of the accounts
*previously stored.
If using full sentences, full stop in the end as well (this goes for everywhere throughout the file)

This goes for all fields regarding `--account`